### DISCUSSION TITLE
World Building / Exploration Meetup 2021-08-06

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#70] [General]
On Friday 2021-08-06 at 12:00 PM PDT we'd like to invite everyone to explore a few virtual worlds as we brainstorm ideas for OMI's virtual home.

The plan is to meet in Discord at noon PDT and check out Jel, other WebXR experiences, and whatever else we come up with. Meanwhile we'd like to take some notes for what platforms we'd like to build virtual OMI spaces for and how best to do so. We'll also be evaluating platforms for our future demo day.

**Comment by LightLodges** [comment]
Worlds to suggest visiting:
I mostly use FrameVR.io or Hubs with clients who need a quick web-based solution to work right now. 
Both teams seem open to future collaborations, especially Mozilla over Virbela for OS dev and interoperability. I personally have worlds in dev in both Spoke and Frame this month. 

I personally liked using New Art City but it may be too cumbersome for our v1.

Engage is picking up steam for edu uses, and Cause mentioned they're working on portability of worlds between Altspace and Engage (Christi recently took on the move of XR Social Club from VRChat to Altspace and has some thoughts/perspective to share on this). 

NeosVR is very open to hybrid and mixed world experiments overall along with Tvoli Cloud and our friends back in Second Life, especially at the Nonprofit Commons. All require downloads and are more heavy/cumbersome on your machines, not WebXR.

Nowhere is AFRAME on the web out of Brooklyn for small group streaming events and would work well for demo days, I have a room there that can host 20 people if anyone wants video meditated events without avatars (people appear as nonagons of their video presence in the AFRAME world). Nowhere generally has some cost to it but has provided me with a free room to use for meetups and testing. While in AFRAME, they're not primarily focused on interoperability; they serve artists & the virtual events industry. https://www.urnowhere.com/
